src/main/index.js

```diff
@@ -4,6 +4,8 @@ const electron = require('electron')
 const app = electron.app
 const ipcMain = electron.ipcMain
 
+const parallel = require('run-parallel')
```

Comment:
Feel free to PR that, but then also remove other places where `run-parallel` is used so it can be removed from `package.json`.